### PR TITLE
Add fun button

### PR DESCRIPTION
**Pull request by BrandonWerbel**
Tell me if you have any more ideas for messages

### CODE REVIEW

**Review by AndrewLester — APPROVED** [review]

File: src/ui.js [hunk 1]
```diff
@@ -71,6 +73,56 @@ cameraRefresh2.addEventListener('click', () => {
...
+        "This message is brought to you by Pizzaria Orso!",
+        "This message is brought to you by Flippn' Pizza!",
+        "This message is brought to you by Baroody Camps!",
+        "This message is brought to you by Dixie!",
+        "Our robot is more structurally sound than our school!",
+        "The librarians hate us, but that's okay!",
+        "Yahweh loves you",
+        "The Fitness Gram Pacer Test is a multistage aerobic capacity test...",
+        "This dashboard has been invaded by goose!",
+        "iBoss knows where you live!",
```
Comment (ErikBoesen) [inline src/ui.js]: I LOVE THIS HAHAHAHA

File: src/ui.js [hunk 2]
```diff
@@ -71,6 +73,56 @@ cameraRefresh2.addEventListener('click', () => {
...
+        "Jesus loves you",
+        "You're a terrible driver!",
+        "Andrew is the only real programmer",
+        "It's not my fault the cameras don't work",
+        "This message is brought to you by Cyclebar!",
+        "This message is brought to you by Pizzaria Orso!",
+        "This message is brought to you by Flippn' Pizza!",
+        "This message is brought to you by Baroody Camps!",
+        "This message is brought to you by Dixie!",
+        "Our robot is more structurally sound than our school!",
```
Comment (ErikBoesen) [inline src/ui.js]: GOLDEN CONTENT

**Comment by ErikBoesen** [comment]
Can we unmerge this PR just so I can have another chance to approve it

**Comment by JosephCarpenter** [comment]
Same here

**Comment by JosephCarpenter** [comment]
We spent the last half an hour yesterday brainstorming more of these while we were waiting for the mechanics to finish with the robot—Brandon is a genius

**Pull request merged**